docs/config.rst

```diff
@@ -386,6 +386,7 @@ The following options can be specified in the Noxfile:
 * ``nox.options.default_venv_backend`` is equivalent to specifying :ref:`-db or --default-venv-backend <opt-default-venv-backend>`.
 * ``nox.options.force_venv_backend`` is equivalent to specifying :ref:`-fb or --force-venv-backend <opt-force-venv-backend>`.
 * ``nox.options.reuse_existing_virtualenvs`` is equivalent to specifying :ref:`--reuse-existing-virtualenvs <opt-reuse-existing-virtualenvs>`. You can force this off by specifying ``--no-reuse-existing-virtualenvs`` during invocation.
+* ``nox.options.no_install`` is equivalent to specifying :ref:`--no-install <opt-no-install>`.
```

Comment:
```suggestion
```

This option cannot be specified in the Noxfile, you're not passing `noxfile=True` to the `Option` constructor. I'd suggest to omit this for now.

Reply:
I was trying to be helpful 🙂

Thanks for the clarification! I didn't know that detail.